include/UtilFunctionTesters/UtilFunctionTester.h

```diff
@@ -5,6 +5,9 @@
 #include "Debug.h"
 #include "Utils.h"
 
+// a purely abstract class to be extended in order to represent
+// the fact that it is a util function that needs to have a test
+// ran
```

Comment:
I'm not sure what "needs to have a test ran" means, or why that fact might be important.